content/en/docs/azure/authentication-oidc.md

```diff
@@ -0,0 +1,209 @@
+    # name of the directory containing your configuration.
+    # For example, your deployment name can be 'my-kubeflow' or 'kf-test'.
+    export KF_NAME=<your choice of name for the Kubeflow deployment>
+
+    # Set the path to the base directory where you want to store one or more
+    # Kubeflow deployments. For example, /opt/.
+    # Then set the Kubeflow application directory for this deployment.
+    export BASE_DIR=<path to a base directory>
+    export KF_DIR=${BASE_DIR}/${KF_NAME}
+
+    # Set the configuration file to use, such as the file specified below:
+    export CONFIG_URI="{{% config-uri-azure-oidc %}}"
+
+    # Generate and deploy Kubeflow:
+    mkdir -p ${KF_DIR}
+    cd ${KF_DIR}
+    kfctl build -V -f ${CONFIG_URI}
+    ```
+
+    * **${KF_NAME}** - The name of your Kubeflow deployment.
+      If you want a custom deployment name, specify that name here.
+      For example,  `my-kubeflow` or `kf-test`.
+      The value of KF_NAME must consist of lower case alphanumeric characters or
+      '-', and must start and end with an alphanumeric character.
+      The value of this variable cannot be greater than 25 characters. It must
+      contain just a name, not a directory path.
+      This value also becomes the name of the directory where your Kubeflow
+      configurations are stored, that is, the Kubeflow application directory.
+
+    * **${KF_DIR}** - The full path to your Kubeflow application directory.
+
+1. Confiure OIDC Auth service settings:
+
+   In `/manifests/stacks/azure/application/oidc-authservice/kustomization.yaml` update settings with values corresponding your app registration.
+
+    ```
+    - client_id=<client_id>
+    - oidc_provider=https://login.microsoftonline.com/<tenant_id>/v2.0
+    - oidc_redirect_uri=https://<load_balancer_ip> or dns_name>/login/oidc
+    - oidc_auth_url=https://login.microsoftonline.com/<tenant_id>/oauth2/v2.0/authorize
+    - application_secret=<client_secret>
+    - skip_auth_uri=
+    - namespace=istio-system
+    - userid-header=kubeflow-userid
+    - userid-prefix=
+    ```
+
+1. Configure OIDC scopes:
+
+    In `/manifests/istio/oidc-authservice/base/statefulset.yaml` update [OIDC scopes](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes) to remove groups and keep profile and email.
+
+    ```
+    - name: OIDC_SCOPES
+     value: "profile email"
+    ```
+
+3. Deploy Kubeflow:
+
+    ```
+    kfctl apply -V -f ${CONFIG_URI}
+    ```
+
+4. Check the resources deployed correctly in namespace `kubeflow`
+
+    ```
+    kubectl get all -n kubeflow
+    ```
+
+## Expose Kubeflow securely over HTTPS
+
+1. Update Istio Gateway to expose port 443 with HTTPS and make port 80 redirect to 443:
+
+    ```
+    kubectl edit -n kubeflow gateways.networking.istio.io kubeflow-gateway
+    ```
+
+    The Gateway spec should look like the following:
+
+    {{< highlight yaml >}}
```

Comment:
Is this highlight yams something website specific? It's not rendered good on GitHub